src/BlobContainer.cpp

```diff
@@ -2,7 +2,6 @@
 // Licensed under the MIT License.
 #include "pch.h"
 #include <dxbcutils.h>
-#include <shaderbinary.h>

```

Comment:
This is the only meaningful change, everything else is the autoformatter being a bit over-zealous. Sorry about that.